### DISCUSSION TITLE
[Feature] Priority options for scanning external libraries

### DISCUSSION THREAD

**Discussion by nodis** [immich-app/immich#4581] [Feature Request]
### The feature

Ps: I don't understand the scanning logic of immich's external image library

1. I have several historical libraries for my collection, such as the 2010 photography photo library. It is impossible to add any new photos to this library, but it will also be scanned when scanning the library, which will cause a huge waste of time. Suggestions are as follows:

I can set the priority or frequency of external image library scanning, for example, for image libraries that take photos on mobile phones, I can set a high priority or specify scanning once an hour, because mobile phones take photos every day, to ensure that the library is updated in a timely manner; For libraries that are not frequently updated, low priority can be set or scanning can be specified once a month or even every six months to maximize efficiency.

thank you

### Platform

- [X] Server
- [x] Web
- [ ] Mobile

**Comment by nodis** [comment]
Does anyone else need this feature?

**Reply by alextran1502** [reply]
somewhat serve the needs here https://github.com/immich-app/immich/pull/4390
